test/io.jl

```diff
@@ -11,22 +11,39 @@ extend this in the original package
 
 using GigaSOM, DataFrames, XLSX, CSV
 
+#create genData and data folder and change dir to dataPath
 cwd = pwd()
+if occursin("jenkins", homedir()) || "TRAVIS" in keys(ENV)
+    genDataPath = mktempdir()
+    dataPath = mktempdir()
+    cd(dataPath)
+else
+    if !occursin("test", cwd)
+        cd("test")
+        cwd = pwd()
+    end
+    dataFolders = ["genData", "data"]
+    for i in dataFolders
+        if !isdir(i)
+            mkdir(i)
```

Comment:
`i` is generally reserved for counters. May you use `dir` or `directory`?

Reply:
changed